pkg/util/kubernetes/node_test.go

```diff
@@ -0,0 +1,38 @@
+/*
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to You under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package kubernetes
+
+import (
+	"context"
+	"fmt"
+	"github.com/apache/camel-k/v2/pkg/client"
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func TestNodeArchLookup(t *testing.T) {
```

Comment:
wonder if this test is better suited to be on an e2e or have a conditional execution as you may not be connected to a k8s cluster and/or you may not have enough privileges.

Reply:
To be removed - it's not needed any more